tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -99,13 +142,13 @@ def _process_resource(self, resource):
             model.reserved = True
 
         if 'size' in self.data:
-            size = self.data.get('size')
+            size = Lookup.extract(self.data.get('size'), resource)
             model.sku = models.SkuDescription()
             model.sku.tier = ResizePlan.get_sku_name(size)
             model.sku.name = size
 
         if 'count' in self.data:
-            model.target_worker_count = self.data.get('count')
+            model.target_worker_count = Lookup.extract(self.data.get('count'), resource)
```

Comment:
If app service SDK has a bug that requires us to always provide SKU and count, unfortunately we need to work around it :(